docs/content/middlewares/forwardauth.md

```diff
@@ -454,7 +459,7 @@ http:
 
 #### `tls.insecureSkipVerify`
 
-TODO add description.
+If `insecureSkipVerify` is `true`, TLS for the connection to authentication server accepts any certificate presented by the server and any host name in that certificate.
```

Comment:
```suggestion
Policy for certificate verification for the secured connection to the authentication server.

If `insecureSkipVerify` is `true`, Traefik accepts any certificate presented by the authentication server and any host name in that certificate.
```

Reply:
It's not related to Traefik, but to auth server.